scripts/exregional_make_grid.sh

```diff
@@ -266,29 +266,36 @@ generation executable (exec_fp):
 # namelist file.
 #
   settings="
-'regional_grid_nml': {
-    'plon': ${LON_CTR},
-    'plat': ${LAT_CTR},
-    'delx': ${DEL_ANGLE_X_SG},
-    'dely': ${DEL_ANGLE_Y_SG},
-    'lx': ${NEG_NX_OF_DOM_WITH_WIDE_HALO},
-    'ly': ${NEG_NY_OF_DOM_WITH_WIDE_HALO},
-    'pazi': ${PAZI},
- }
```

Comment:
Several of these blocks show up, and this is to ensure that the resulting input file for the tool is proper YAML.